Script to predict, interface for the API

renanmouraf opened the pull request.

renanmouraf committed:
predict script and fix on train.py to index false when saving csv

renanmouraf committed:
main function

cpatrickalves reviewed (changes requested):
@renanmouraf the PKL and CSV outputs from **train_model.py** should be placed in the **models** root folder.
I think you are saving them at src/models/, which should contain only the python scripts

renanmouraf committed:
fixing paths for output files

renanmouraf commented:
Done

cpatrickalves committed:
Refactoring:
- move load_model function to the HousePriceModel class
- Change the parameters of the model
- Saves de model inside a folder in models/
- create methods to get examples and input features of the model
- Add some prints to show the progress
- Update the export_model function to save in de model folder

cpatrickalves reviewed (approved):
Igor said that is ok
